_field-types/supported-field-types/knn-vector.md

```diff
@@ -22,19 +22,19 @@ PUT test-index
 {
   "settings": {
     "index": {
-      "knn": true,
-      "knn.algo_param.ef_search": 100
+      "knn": true
     }
   },
   "mappings": {
     "properties": {
       "my_vector": {
         "type": "knn_vector",
         "dimension": 3,
+        "space_type": "l2",
+        "mode": "in_memory",
+        "compression_level": "2x",
```

Comment:
why we are putting a 2x as default compression here?

Reply:
I figured Id show all the parameters and what they look like